CHANGELOG.md

```diff
@@ -10,6 +10,105 @@
+  ([#13945](https://github.com/getsentry/sentry-javascript/pull/13945))**
+
+There is no need to add the Node `--import` flag anymore because the SDK now wraps your application with a dynamic
+`import()`. Please update your start command to avoid initializing Sentry twice (BREAKING CHANGE). If you prefer to use
+the `--import` flag, you can enable this option in your `nuxt.config.ts`:
+
+```js
+sentry: {
+  dynamicImportForServerEntry: false;
+}
+```
+
+- **feat(nuxt): Respect user-provided source map generation settings
+  ([#14020](https://github.com/getsentry/sentry-javascript/pull/14020))**
+
+Source maps As Nuxt sets `sourcemap.client: false`, it's not possible to determine whether this setting was set by you
+or the framework. Set this manually like this:
+
+```js
+export default defineNuxtConfig({
+  sourcemap: {
+    client: true,
+  },
+});
+```
+
+- feat(nuxt): Log server instrumentation might not work in dev
+  ([#14021](https://github.com/getsentry/sentry-javascript/pull/14021))
+- feat(nuxt): Add Http `responseHook` with `waitUntil`
+  ([#13986](https://github.com/getsentry/sentry-javascript/pull/13986))
+
+### Important Changes
+
+- **feat(vue): Add Pinia plugin ([#13841](https://github.com/getsentry/sentry-javascript/pull/13841))**
+
+Support for [Pinia](https://pinia.vuejs.org/) is added in this release for `@sentry/vue`. To capture Pinia state data,
+add `createSentryPiniaPlugin()` to your Pinia store:
+
+```javascript
+import { createPinia } from 'pinia';
+import { createSentryPiniaPlugin } from '@sentry/vue';
+
+const pinia = createPinia();
+
+pinia.use(createSentryPiniaPlugin());
+```
+
+- **feat(node): Implement Sentry-specific http instrumentation
+  ([#13763](https://github.com/getsentry/sentry-javascript/pull/13763))**
+
+This change introduces a new `SentryHttpInstrumentation` to handle non-span related HTTP instrumentation, allowing it to
+run side-by-side with OTel's `httpInstrumentation`. This improves support for custom OTel setups and avoids conflicts
+with Sentry's instrumentation. Additionally, the `spans: false` option is reintroduced for `httpIntegration` to disable
+span emission while still allowing custom `HttpInstrumentation` instances (`httpIntegration({ spans: false })`).
+
+- **feat(core): Make stream instrumentation opt-in
+  ([#13951](https://github.com/getsentry/sentry-javascript/pull/13951))**
+
+This change adds a new option `trackFetchStreamPerformance` to the browser tracing integration. Only when set to `true`,
+Sentry will instrument streams via fetch.
+
+### Other Changes
+
+- feat(node): Expose `suppressTracing` API ([#13875](https://github.com/getsentry/sentry-javascript/pull/13875))
+- feat(replay): Do not log "timeout while trying to read resp body" as exception
+  ([#13965](https://github.com/getsentry/sentry-javascript/pull/13965))
+- chore(deps): bump astro from 4.1.1 to 4.16.1 in /dev-packages/e2e-tests/test-applications/cloudflare-astro
```

Comment:
Should we remove these?

Reply:
Yeah true, the version bumps for test-related stuff is not relevant here.